### PR TITLE
sonarqube-quality-gate 0.0.2

### PR DESCRIPTION
**Pull request by abonham**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by adborbas** [comment]
Hi @abonham , could you please sign the CLA?

**Comment by abonham** [comment]
Signed CLA!

File: steps/sonarqube-quality-gate/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+inputs:
+- opts:
+    description: |
+      API token from sonar qube provider
+    is_expand: true
+    is_required: true
+    summary: API token from sonar qube provider
+    title: SonarQube Token
+    value_options: []
+  sonar_token: $SONAR_TOKEN
```
Comment (trapacska) [inline steps/sonarqube-quality-gate/0.0.2/step.yml:46]: this input should be marked as sensitive: https://devcenter.bitrise.io/contributors/create-your-own-step/#sensitive-inputs

File: steps/sonarqube-quality-gate/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- opts:
+    is_required: true
+    summary: Sonarcloud organisation key
+    title: Organisation Key
+  organisation_key: null
```
Comment (trapacska) [inline steps/sonarqube-quality-gate/0.0.2/step.yml:51]: maybe this is also sensitive

**Review by trapacska — CHANGES_REQUESTED** [review]

**Comment by sylank** [comment]
@abonham This is a friendly ping message. I am just wondering if you mind fixing the review comments? 

:v:

**Comment by adborbas** [comment]
Hi @abonham, I'll close this PR due to inactivity. Please feel free to reopen it incase you want to.

**Pull request closed without merging**